frontend/src/main/web/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Router from 'react-router';
+import RootContent from './lib/components/RootContent';
+import Views from './lib/constants/Views.js';
+import Configs from './lib/constants/Configs';
+import StringUtils from './lib/utils/StringUtils';
+import _ from 'lodash';
+import css from './index.css'
+
```

Comment:
Looks like this file's job is to initiate the React router using some data found in some attributes of `#main-content`. Please add a doc comment that says that.